packages/remark-stringify/types/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// TypeScript Version: 3.0
+
+declare module 'remark-stringify' {
```

Comment:
Use `declare namespace`

Reply:
It's a different issue. There are just two types to define types. One is using `declare module 'X'` and another one is using `declare namespace X`. The one I prefer is used by React and Angular. (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L48, https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/angular/index.d.ts#L33)  I just want to keep the consistency that we're using now.

And Babel won't read type definition files. 😄